CityOfDetroit/bloom
# Fix/features filtering

## Issue

- Closes #901

## Description

A few fixes:
 * The comparison needed to be "="
 * We needed to specify `listing_features` when mapping filter parameters to db queries
 * We needed to add `listing_features` to the set of tables that are joined and selected for full listing views

Also adds a coupla tests to verify stuff works.

## Type of change


## How Can This Be Tested/Reviewed?

Reseed the db, deploy locally, and navigate to http://localhost:3000/listings/filtered?page=1&elevator=true. You should see 8 listings.

## Checklist:
